### DISCUSSION TITLE
I am using ROS-noetic for O3D303 and O3X100. If the ifm3d version is 0.20.x, what is the compatible ifm3d_ros version?

### DISCUSSION THREAD

**Discussion by coa06yl** [ifm/ifm3d-ros#71] [General]
I am using ROS-noetic for O3D303 and O3X100. If the ifm3d version is 0.20.x, what is the compatible ifm3d_ros version?

**Comment by lola-masson** [comment]
Hi @coa06yl, 

For O3D and O3X, the last compatible version of the ifm3d-ros wrapper is 0.7.0, which should work with ifm3d 0.20.3. Unfortunately it has not gotten an update for a while, but we are working on bringing O3X and O3D compatibility to the newer versions of ifm3d, which will be a first step before updating the ROS wrappers to support these devices as well.
